src/main/java/com/yas/controller/CustomerController.java

```diff
@@ -14,32 +14,27 @@
 import com.yas.dao.ServicesRepository;
 import com.yas.model.Customer;
 import com.yas.model.Services;
+import com.yas.service.CustomerService;
 
 @RestController
 public class CustomerController {
-	
-	@Autowired
-	CustomerRepository cr;
-	
+
 	@Autowired
-	ServicesRepository sr;
-	
+	CustomerService cs;
 
 	@PostMapping("/add-customer")
 	public Customer addCustomer(@RequestBody Customer custom) {
-		List<Services> listServe = sr.findAll();
-		custom.setServe(listServe);
-		return cr.save(custom);
+		return cs.addCustomer(custom);
 	}
-	
+
 	@GetMapping("/get-customer")
-	public List<Customer> getCustomer(){
-		return cr.findAll();
+	public List<Customer> getCustomer() {
+		return cs.getCustomer();
 	}
-	 
+
```

Comment:
plz change the annotation

Reply:
I have added the annotation please check it .